db/data-generators/generate-relational-data.js

```diff
@@ -0,0 +1,101 @@
+// this file creates 4 csvs that are stored in the local data folder
+// use the postgres loader files to load the csv into a db
+const fs = require('fs');
+const colorList = require('./colors.js');
+
+const getRandomInt = (min, max) => Math.floor(Math.random() * ((max - min) + 1)) + min;
+
+// generate products
+console.log('starting products');
+
+const text = `mumblecore migas semiotics
```

Comment:
could probably factor this out into helper file since they are very similar for both types of databases